doc/index.rst

```diff
@@ -0,0 +1,164 @@
+:orphan:
+
+Matplotlib is a Python 2D plotting library which produces publication quality
+figures in a variety of hardcopy formats and interactive environments across
+platforms.  Matplotlib can be used in Python scripts, the Python and IPython_
+shells, the Jupyter_ notebook, web application servers, and four graphical user
+interface toolkits.
+
+.. _IPython: http://ipython.org
+.. _Jupyter: http://jupyter.org
+
+.. raw:: html
+
+   <div class="responsive_screenshots">
+      <a href="tutorials/introductory/sample_plots.html">
+         <div class="responsive_subfig">
+         <img align="middle" src="_images/sphx_glr_membrane_thumb.png"
+          border="0" alt="screenshots"/>
+         </div>
+         <div class="responsive_subfig">
+         <img align="middle" src="_images/sphx_glr_histogram_thumb1.png"
+          border="0" alt="screenshots"/>
+         </div>
+         <div class="responsive_subfig">
+         <img align="middle" src="_images/sphx_glr_contour_thumb.png"
+          border="0" alt="screenshots"/>
+         </div>
+         <div class="responsive_subfig">
+         <img align="middle" src="_images/sphx_glr_3D_thumb.png"
+          border="0" alt="screenshots"/>
+         </div>
+      </a>
+   </div>
+   <span class="clear_screenshots"></span>
+
+Matplotlib tries to make easy things easy and hard things possible.  You
+can generate plots, histograms, power spectra, bar charts, errorcharts,
+scatterplots, etc., with just a few lines of code.  For examples, see the
+:doc:`sample plots <tutorials/introductory/sample_plots>` and :doc:`thumbnail
+gallery <gallery/index>`.
+
+For simple plotting the `pyplot` module provides a MATLAB-like interface,
+particularly when combined with IPython.  For the power user, you have full
+control of line styles, font properties, axes properties, etc, via an object
+oriented interface or via a set of functions familiar to MATLAB users.
+
+Installation
+------------
+
+Visit the :doc:`Matplotlib installation instructions <users/installing>`.
+
+Documentation
+-------------
+
+This is the documentation for Matplotlib version |version|.
+
+To get started, read the :doc:`User's Guide <users/index>`.
+
+.. raw:: html
+
+   <?php include('http://matplotlib.org/versions.html');?>
```

Comment:
So, does this work when uploaded?

Reply:
ping @tacaswell

Reply:
As far as I know its being served from Github pages and that does not support any server side dynamic pages https://www.quora.com/How-can-I-run-my-PHP-content-on-GitHub-pages